cpu/cortexm_common/include/cpu.h

```diff
@@ -25,13 +25,12 @@
  * @author      Joakim Gebart <joakim.gebart@eistec.se>
  */
 
-#ifndef CORTEXM_COMMON_H_
-#define CORTEXM_COMMON_H_
-
+#ifndef CPU_H_
+#define CPU_H_
 
 #include "cpu_conf.h"
 
-/**
+/*
  * TODO: remove once core was adjusted
```

Comment:
yes, there is: https://github.com/RIOT-OS/RIOT/issues/1892

Once in a time, boards/cpus used `dINT` and `eINT` macros for enabling/disabling interrupts. Then the current API with `enableIRQ` and `disableIRQ` was introduced. But there are still some occurances of the old calls left, so we were not able to remove them so far...